sass/components/_todo.scss

```diff
@@ -0,0 +1,13 @@
+.todo {
+  background-color: hsla(50, 50, 50, 0.3);
+  border: double hsla(50, 75, 50, 0.5) 6px;
```

Comment:
Just commentary, I don't want to block on a subjective matter in a non-user-facing context.

I personally find this double border a bit funky. I think it would be nice to have a consistent "callout" style for these sorts of things with styling similar to what was done for the "bad code blocks."

A few css classes that we could use to produce these sorts of blocks would be handy.
<img width="833" alt="image" src="https://github.com/bevyengine/bevy-website/assets/200550/3e436078-439c-449d-a8f9-6140c3399166">